Components/Grid/grid.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>UI-Simple-Grid</title>
+    <link rel = "stylesheet" href = "/style.css">
+    <link rel = "stylesheet" href = "grid.css">
```

Comment:
Having only one CSS file is better for the loading-time of your pages, as it means less HTTP requests.

Reply:
I will keep that in mind ..and also as you have suggested using alt in my project.Thankyou!